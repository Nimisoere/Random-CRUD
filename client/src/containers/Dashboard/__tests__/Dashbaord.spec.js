import React from "react";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import DashboardComponent from "../Dashboard";

Enzyme.configure({ adapter: new Adapter() });

function setup() {
  const enzymeWrapper = shallow(<DashboardComponent />);
  return {
    enzymeWrapper
  };
}

describe("components", () => {
  describe("DashboardComponent", () => {
    it("Match Snapshot", () => {
      const { enzymeWrapper } = setup();
      expect(enzymeWrapper).toMatchSnapshot();
    });
  });
});
