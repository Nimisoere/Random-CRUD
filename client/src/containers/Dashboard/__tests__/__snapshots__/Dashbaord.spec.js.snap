// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components DashboardComponent Match Snapshot 1`] = `ShallowWrapper {}`;
